### PR TITLE
feat: add text style to textarea

### PR DESCRIPTION
**Pull request by boyarskiy**
References [link the ticket here]

## Motivation and context

In order to change text style for textarea component, we have to add textStyle props.

## Before

N/A

## After

textStyle prop is available

## How to test

[Add a deep link and instructions how to verify the new behavior]

### CODE REVIEW

File: src/components/textarea/index.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ interface Props {
   onChange?: ChangeEventHandler<HTMLTextAreaElement>;
   rows?: number;
   cols?: number;
+  textStyle?: string;
```
Comment (GutierrezGo) [inline src/components/textarea/index.tsx:21]: Do we want to pass all the typography tokens or the fontStyle in the texarea are restricted to some styles?

Comment (GutierrezGo) [reply src/components/textarea/index.tsx:21]: I don't think we need heading, button and some other textStyle tokens. Would you mind doble checking with the Designer? Thank you 🌸

Comment (boyarskiy) [reply src/components/textarea/index.tsx:21]: I do not know if there are some restrictions, based on a talk with designer usually there will be `body` and `body-small`. Maybe it makes sense to specify just those two

Comment (GutierrezGo) [reply src/components/textarea/index.tsx:21]: I will pass only the textStyles for its future use/case

**Review by GutierrezGo — APPROVED** [review]
Thank you 🌸

**Pull request merged**